scripts/load_bq.py

```diff
@@ -14,7 +14,7 @@
 from google.api_core import retry, exceptions
 
 from util import run
-from util import cfg
+from util import cfg, create_client_options, ApiEndpoint
```

Comment:
I would advice agains `from A import B` without good reason. The prime concern is that such import prevents mocking in tests.

Reply:
Done.